integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java

```diff
@@ -344,6 +349,181 @@ public static Domain createDomainResourceWithAuxiliaryImage(
     return domainCR;
   }
 
+  /**
+   * Create a domain object for a Kubernetes domain custom resource using the basic WLS image
+   * and MII auxiliary images containing the doamin or/and cluster configuration.
+   *
+   * @param domainResourceName name of the domain resource
+   * @param domNamespace Kubernetes namespace that the domain is hosted
+   * @param baseImageName name of the base image to use
+   * @param adminSecretName name of the new WebLogic admin credentials secret
+   * @param repoSecretName name of the secret for pulling the WebLogic image
+   * @param encryptionSecretName name of the secret used to encrypt the models
+   * @param replicaCount number of managed servers to start
+   * @param clusterNames a list of the cluster name to add auxiliary image in domain
+   * @param auxiliaryImagePathVolume a map of auxiliary image path, parent location for Model in Image model
+   *                                 and WDT installation files as the key and a list of auxiliary image volume names
+   *                                 as the values for the key
+   * @param auxiliaryImageDomainScopeNames a list of image names including tags, image contains the domain model,
+   *                                       application archive if any and WDT installation files
+   * @param auxiliaryImageClusterScopeNames a list of images containing the files to
+   *                                        config cluster scope auxiliary image
+   * @return domain object of the domain resource
+   */
+  public static Domain createDomainResourceWithAuxiliaryImageClusterScope(
+      String domainResourceName,
+      String domNamespace,
+      String baseImageName,
+      String adminSecretName,
+      String repoSecretName,
+      String encryptionSecretName,
+      int replicaCount,
+      List<String> clusterNames,
+      Map<String, List<String>> auxiliaryImagePathVolume,
+      List<String> auxiliaryImageDomainScopeNames,
+      List<String> auxiliaryImageClusterScopeNames) {
+
+    Domain domainCR = null;
+    int i = 0;
+
+    for (String clusterName : clusterNames) {
```

Comment:
Instead of calling the `createDomainResourceWithAuxiliaryImageClusterScope` in the for loop, it might be more clear to pass `clusterNames` as an argument to this method and then iterate over the `clusterNames` list in that method.

Reply:
there are two createDomainResourceWithAuxiliaryImageClusterScope, one take one cluster, another one takes a list of cluster. This test only used createDomainResourceWithAuxiliaryImageClusterScope with one cluster. createDomainResourceWithAuxiliaryImageClusterScope with a cluster list(multiple cluster) is NOT used by this changes but saved for later tests when we need to config cluster scope configurations in two cluster